index.html

```diff
@@ -46,8 +46,8 @@ <h1>This is my Ethersjs Demo!</h1>
   </div>
 
   <script>
-    window.ethereum.enable()
-    var provider = new ethers.providers.Web3Provider(web3.currentProvider, 'ropsten');
+   window.eth_requestAccounts;
```

Comment:
does this work? Have you tried it?

Reply:
yes I have tried infact i have solved it for 2-3 more people . their code had no other error . even tough this shows as warning in console but as soon as you change it the code starts working.